NEWS.md

```diff
@@ -1,3 +1,8 @@
+### jemmer 0.0.11
+ * VIMC-1380 method 2 impact calculation  
+ * R/impact_method2.R - this is the impact calculation functions; currently read from .csv recipe; will be changed when recipe is imported montagu
```

Comment:
Typically NEWS files focus on the user-focussing bits of the changes, rather than the implementation details (here the names of the files that you're using)

Reply:
Done.